hi-ml-cpath/requirements_run.txt

```diff
@@ -1,6 +1,7 @@
 coloredlogs==15.0.1
 cucim==22.04.00
 girder-client==3.1.14
+hi-ml>=0.2.3
```

Comment:
Is there a reason this is lower bounded here but pinned in environment.yml?

Reply:
`environment.yml` is a locked environment, similar to what we have now in IE-DL: all dependencies and secondary are pinned. To update, we use a script - see header at the top of the env file.